guidance.md

```diff
@@ -1 +1,95 @@
+
+One unintended consequence of this guidance is that services either don't introduce new API versions because they believe their change is not breaking (it would be a minor version per semantic versioning rules). 
+
+> For a recent example, see https://github.com/Azure/azure-rest-api-specs/pull/8585
+
+Another unintended consequence is that since clients are shielded behind an API version, service teams do not distinguish between making *changes that require a new API version* and *contract or semantical breaking changes*. Overestimating the value of changes and underestimating the cost for customers to adopt *contract or semantical breaking changes* is a very common tendency among product teams - not just in Azure. The .NET Framework compatibility council has many years of experience of communicating this fact to partner teams.
+
+> TODO: Insert examples here
+
+### Issues with the current service API versioning scheme
+
+From a customer's perspective, a date based versioning scheme does not carry any semantic information as to how large the change between two API versions is. I have anecdotal evidence that the fact a developer cannot easily determine if a change is large or trivial is cause for dissatisfaction.
+
+> NOTE: This is primarily an issue where we have "contractual or semantical breaking changes" between api versions, since if the only changes we introduce even between service api versions are additive, there is no need to signal larger changes between API versions.
+
+### Client library specific issues with the current service API guidance
+
+The current official versioning guidelines state that each new API version corresponds to a new major version of a client library. This has several challenges:
+
+- Many languages/runtimes are unable to load multiple versions of the same library in the same process.
+- New major library versions cause great disruption in those ecosystems and should be avoided. One of the most common issues is the diamond reference problem where all libraries have to agree on the one and only version they reference.
+- For language ecosystems where semantic versioning is used for library versioning, the bar for what a semantic major version is differs from the bar for "changes that require a service api version update". Many "changes that require a service api version update" are minor version changes according to semantic versioning.
+- Publishing packages with the version number in the package name causes an explosion of the number of packages. This impacts how easy it is to find the correct package for a given service (the package manager's search engine generally boost the most commonly downloaded package, which means that new versions are effectively hidden from search). It also prevents tooling from helping you upgrade versions since tools don't understand that the different packages can be treated as different versions of the same package. 
```

Comment:
This is a bit confusing. Are you talking about an option for our first 4 languages, or the fact that Jeff wants Go, C, C++ to version sxs? i.e. we are not renaming packages when we version them, are we?